dds/DCPS/ReceivedDataElementList.h

```diff
@@ -20,10 +20,6 @@
 
 #include "dds/DdsDcpsInfrastructureC.h"
 
-#if !defined (ACE_LACKS_PRAGMA_ONCE)
```

Comment:
Why was this removed?  Seems like we have it in many places, see PR #897 for example.